### ISSUE TITLE
 websocket request blocked by cors policy in 5.2

### ISSUE THREAD

**Issue by chulman** [spring-projects/spring-framework#23904]
**Affects:** 5.2.0

Implemented a websocket configurer to allow requests from all origins, but response header does not include the `Access-Control-Allow-Origin: *`

**Code to:**
```java
import org.springframework.context.annotation.Configuration;
import org.springframework.web.socket.config.annotation.EnableWebSocket;
import org.springframework.web.socket.config.annotation.WebSocketConfigurer;
import org.springframework.web.socket.config.annotation.WebSocketHandlerRegistry;

@Configuration
@EnableWebSocket
public class WebSocketConfig implements WebSocketConfigurer {
    @Override
    public void registerWebSocketHandlers(WebSocketHandlerRegistry registry) {
        registry.addHandler(new WsTranportHandler(), "/ws").setAllowedOrigins("*").withSockJS();
    }
}
```

It works in spring-websocket-5.1.10

**Comment by rstoyanchev** [comment]
@chulman this is very likely a duplicate of #23843. Can you try with `5.2.1.BUILD-SNAPSHOT` via https://repo.spring.io/snapshot please?

**Comment by chulman** [comment]
I tested 5.2.1.build-snapshot and it works. It seems to be a problem with webmvc-5.2.0.release.

thanks.
